kogito-editors-js/packages/boxed-expression-component/src/__tests__/components/Table/Table.test.tsx

```diff
@@ -238,46 +239,11 @@ describe("Table tests", () => {
 
       expect(mockedOnColumnUpdate).toHaveBeenCalled();
       expect(mockedOnColumnUpdate).toHaveBeenCalledWith(
-        [{ label: newColumnName, accessor: newColumnName, dataType: DataType.Boolean } as ColumnInstance],
+        [{ label: newColumnName, accessor: columnId, dataType: DataType.Boolean } as ColumnInstance],
         undefined,
         0
       );
     });
-
-    test("should trigger onRowsUpdate, when changing the column name, via popover, of a column with a value in an existing row", async () => {
```

Comment:
@vpellegrino Did you remove this because of the regression you found?

Reply:
Because with the introduction of ids for the column, the `onRowsUpdate` is not triggered anymore.
Previously, a row object were containing, as keys, the names of the column. So when renaming a column, there was the need of updating the whole rows collection.
Now the row object keys are the columns ids, that do not change after renaming.